keps/2076-kueuectl/README.md

```diff
@@ -0,0 +1,342 @@
+its namespace selector matches to LocalQueue's namespace.
+
+Format:
+
+```
+kueuectl create lq|localqueue lqname
+–-namespace=namespace                  # uses context's default namespace if not specified
+–-clusterqueue=cqname
+--skip-validation
+```
+
+Output:  
+A simple confirmation `localqueue.kueue.x-k8s.io/xxxxx created`
+
+### List ClusterQueue
+
+List all ClusterQueues, potentially limiting output to those that are active/inactive and 
+matching the label selector. Format:
+
+```
+kueuectl list cq|clusterqueue(s)
+--active=*|true|false
+--selector=selector                   # label selector
+```
+
+Output columns:
+
+* Name
+* Cohort
+* Pending Workloads
+* Admitted Workloads
+* Status (active or not)
+* Age
+
+### List LocalQueue 
+
+Lists LocalQueues that match the given criteria: point to a specific CQ, 
+being active/inactive, belonging to the specified namespace or matching the label
+selector.
+Format: 
+
+```
+kueuectl list lq|localqueue(s) 
+–-namespace=ns                        # uses context's default namespaces if not specified
+--all-namespaces | -A
+-–clusterqueue=clusterqueue 
+–-active=*|true|false
+--selector=selector                   # label selector
+```
+
+Outputs columns:
+
+* Namespace (if -A is used)
+* Name
+* ClusterQueue
+* Pending Workloads 
+* Admitted Workloads 
+* Status (active or not)
+* Age
+
+### List Workloads
+
+Lists Workloads that match the provided criteria. Format:
+
+```
+kueuectl list workloads 
+--namespace=ns                        # uses context's default namespace if not specified
+--all-namespaces | -A
+--clusterqueue=cq 
+-–localqueue=lq 
+-—only-pending
+—-only-admitted
+--selector=selector
+```
+
+Output:
+
+* Namespace (if -A is used)
+* Workload name
+* CRD type (truncated to 10 chars)
+* CRD name
+* LocalQueue
+* ClusterQueue
+* Status
+* Position in Queue (if Pending)
+* Age
+
+
+### Stop ClusterQueue
+
+Stops admission and execution inside the specified ClusterQueue, possibly
+limiting the action only to the selected ResourceFlavor.
+Format:
+
+```
+kueuectl stop clusterqueue|cq cqname
+--keep-already-running
+--resource-flavor=rfname
+```
+
+Output:
+None.
+
+### Resume ClusterQueue
+
+Resumes admission inside the specified ClusterQueue.
+Format:
+```
+kueuectl resume clusterqueue|cq cqname 
+-–resource-flavor=rfname
+```
+Output:
+None.
+
+### Stop LocalQueue
+
+Stops execution (or just admission) of Workloads coming from the given LocalQueue. 
+This requires adding StopPolicy to LocalQueue and enforcing its changes in ClusterQueue.
+Format:
+```
+kueuectl stop localqueue|lq lqname
+--keep-already-running
+```
+Output:
+None.
+
+
+### Resume LocalQueue
+
+Resumes admission of Workloads coming from the given LocalQueue.
+Format:
+```
+kueuectl resume localqueue|lq lqname 
+```
+Output:
+None.
+
+### Stop Workload
+
+Puts the given Workload on hold. The Workload will not be admitted and 
+if it is already admitted it will be put back to queue just as if it was preempted. 
```

Comment:
Clarify that this can use the `.spec.active` Field.

Reply:
Done.